app/controllers/location.js

```diff
@@ -0,0 +1,55 @@
+const https = require("follow-redirects").https;
+require("dotenv").config();
+
+const placeDetails = function() {
+    this.places = [];
+};
+
+//Declare loction information
+const googleApiKey = process.env.GOOGLE_API;
+const latitude = 52.361778;
+const longitude = 4.907370;
+const radius = 500;
+
+
+//Find places within the specified radius, based on the coordinates provided.
+function placeSearch(latitude, longitude, radius) {
+    https.request({
+        host: "maps.googleapis.com",
+        path: "/maps/api/place/nearbysearch/json?location=" + latitude + "," + longitude + "&radius=" + radius + "&type=bar&key=" + googleApiKey,
+        method: "GET"
+    },
+        PlaceResult).end();
+}
+
+//Place retrieved data in an array, and display data in terminal (for now)
+function PlaceResult(response) { 
+    let p;
+    let data = "";
+    let sdata = "";
+    const PD = new placeDetails();
+
+    response.on("data", (chunk) => {
+        data += chunk;
+    });
+    response.on("end", () => {
+        sdata = JSON.parse(data);
+        if (sdata.status === "OK") {
+            console.log("Status: " + sdata.status);
+            console.log("Results: " + sdata.results.length);
+            for (p = 0; p < sdata.results.length; p++) {
```

Comment:
Why don't you use `i` as the index?

Reply:
ah i don't know, changed back to I as index